tests/plugins/link/emailwithquestionmark.js

```diff
@@ -0,0 +1,19 @@
+/* bender-tags: editor, link, email */
```

Comment:
Previous tests concerning `email link` where quite messy distributed in files with names  `mail_link` and `mail_link2`. That was the reason why I decided to put new test to a new file, but probably it would be better to just clean up a bit which I did. Still the tests concerning `emailProtection` seem to be better kept in separate files, so I moved the one unrelated test to `link` file and there will also be the other tests you mentioned in the comments.
I just wonder how having a one big file (with over 40 test cases) affects Bender's performance - is it better than a few smaller files?

Reply:
👍 

> I just wonder how having a one big file (with over 40 test cases) affects Bender's performance - is it better than a few smaller files?

I think visible performance difference may start somewhere near 1000 test cases in one file vs 1000 test files. One test files should perform better as each smaller file needs to be loaded which takes time.